projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -6498,15 +6705,19 @@ public string MotherTransferredHelper
             {
                 // IJE values are Y, N, U, only set to "hosp-trans" if value is Y
                 // https://build.fhir.org/ig/HL7/fhir-bfdr/usage.html#mother-or-infant-transferred
-                if (value == "Y")
+                if (value == "Y" || value == "U")
```

Comment:
Should this `value == "U"` check be here? It seems like the comment says that it should be set to "hosp-trans" only if the value is "Y". There's also a `value == "U"` check on line 6716, I'm not sure if that's an intentional continuation of the logic.